app/src/main/crypto.ts

```diff
@@ -225,6 +231,28 @@ export class Crypto {
     });
   }
 
+  /**
+   * Encrypts a message to a source given the source public key. Message
+   * will be encrypted to source and journalist public keys.
+   * @param plaintext - The message plaintext
+   * @param sourcePublicKey - Source PGP public key
+   * @returns Promise<string> - The encrypted ciphertext
+   */
+  async encryptSourceMessage(
+    plaintext: string,
+    sourcePublicKey: string,
+  ): Promise<string> {
+    if (!this.journalistPublicKey) {
```

Comment:
I think we should make this required instead of having it be optional and failing here, i.e. fail in Config.load() or Config.readVar().